client.go

```diff
@@ -109,15 +109,20 @@ func (c *Client) invokePlugin(ctx context.Context, name string, r *types.Request
 
 	out, err := cmd.Output()
 	if err != nil {
+		msg := name
+		if len(out) > 0 {
+			// include out only if it has content
+			msg = fmt.Sprintf("%s: %s", name, out)
+		}
 		// ExitError is returned when there is a non-zero exit status
 		if _, ok := err.(*exec.ExitError); ok {
 			var pe types.PluginError
 			if err := json.Unmarshal(out, &pe); err != nil {
-				return nil, errors.Wrapf(err, "%s: %s", name, out)
+				return nil, errors.Wrap(err, msg)
```

Comment:
Is it possible to include status code here too?

Reply:
should we also have a check `if len(out) > 0` then add it to the output?

Reply:
Added the exit code as well as enhanced the capturing of stdout and stderr from plugin invocation